src/sentry/api/endpoints/sentry_internal_app_token_details.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+from rest_framework import status
+from django.http import Http404
+
+from sentry.api.bases import (
+    SentryInternalAppTokenPermission, SentryAppBaseEndpoint,
+)
+from sentry.models import ApiToken
+from sentry.features.helpers import requires_feature
+from sentry.mediators.sentry_app_installation_tokens import Destroyer
+
+
+class SentryInternalAppTokenDetailsEndpoint(SentryAppBaseEndpoint):
+    permission_classes = (SentryInternalAppTokenPermission, )
+
+    def convert_args(self, request, sentry_app_slug, token, *args, **kwargs):
+        # get the sentry_app from the SentryInternalAppTokenDetailsEndpoint class
```

Comment:
did you mean "get the sentry_app from the SentryAppBaseEndpoint" ?

Reply:
@MeredithAnya Yes :)